src/cargo/ops/cargo_generate_lockfile.rs

```diff
@@ -36,6 +36,20 @@ pub fn generate_lockfile(ws: &Workspace<'_>) -> CargoResult<()> {
 }
 
 pub fn update_lockfile(ws: &Workspace<'_>, opts: &UpdateOptions<'_>) -> CargoResult<()> {
+    if opts.aggressive && opts.to_update.is_empty() {
```

Comment:
I would suggest adding a comment to tell others this is in a transition period.

Reply:
I think `this will become a hard error in a future release.` already indicates this is in a transition period. Do you have any suggestions on what message should we warn here?

Reply:
Our other warnings during the transition period don't say anything else. Maybe we can keep it consistent?

Reply:
I mean that adding code comment to help other developers know what the next step is and when to do it. It's fine without that, tho. Not a big deal.

Reply:
Oh, get it. I'll add it.

Reply:
Added. 
Thanks for your review!  💚 💙 💜 💛 ❤️